content/pt-br/docs/concepts/overview/working-with-objects/kubernetes-objects.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Entendendo os objetos do Kubernetes
+content_type: concept
+weight: 10
+card:
+ name: concepts
+ weight: 40
+---
+ 
+<!-- overview -->
+Esta página explica como os objetos do Kubernetes são representados na API do Kubernetes e como você pode expressá-los no formato `.yaml`.
+ 
+<!-- body -->
+## Entendendo os objetos do Kubernetes  {#kubernetes-objects}
+ 
+*Os objetos do Kubernetes* são entidades persistentes no sistema do Kubernetes. Usuários do Kubernetes essas entidades para representar o estado do cluster. Especificamente, eles podem descrever:
```

Comment:
This phrase is strange: "Usuários do Kubernetes essas entidades para representar". Can you rephrease it please?